### PR TITLE
Fix doc 

### PR DESCRIPTION
**Pull request by liutaohua**
In the course of reviewing the documentation, I found many obvious problems that needed to be fixed.

I also found that many Chinese documents remained comming soon for a long time, and I added some translations in this pr.

Checking documents and translating documents is a very eye-testing job. Now I feel I am going blind, so I end this work on documents, hoping to help the community.

### CODE REVIEW

**Comment by jixuan1989** [comment]
Hi, you can try:
`git merge origin/master`
and then,
`git cherry-pick eb26b51c9467f3234f8d09fab4659f912f488feb`

File: docs/Documentation-CHN/UserGuide/0-Get Started/2-Frequently asked questions.md [hunk 1]
```diff
@@ -19,6 +19,149 @@
...
+## 在哪里可以找到 IoTDB 的数据文件？
+
+在默认的设置里，数据文件（包含 TsFile，metadata，WAL）被存储在```IOTDB_HOME/data```文件夹。
+
+## 如何知道 IoTDB 中存储了多少时间序列？
+
+使用 IoTDB 的命令行接口:
+
+```
+IoTDB> show timeseries root.*
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/0-Get Started/2-Frequently asked questions.md]: 跟下边统一一下吧，都不带 *，英文的也去掉 *

File: docs/Documentation-CHN/UserGuide/0-Get Started/2-Frequently asked questions.md [hunk 2]
```diff
@@ -19,6 +19,149 @@
...
+
+在 IoTDB 的命令行接口中使用 SQL ```SHOW TIMESERIES <timeseries path>```:
+
+例如：如果您想知道所有 timeseries 的类型 \<timeseries path> 应该为 `root`。上面的 SQL 应该修改为:
+
+```
+IoTDB> show timeseries root
+```
+
+如果您想查询一个指定的 sensor, 您可以修改 \<timeseries path> 为 sensor 的名字。比如:
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/0-Get Started/2-Frequently asked questions.md]:
```suggestion
如果您想查询一个指定的时间序列, 您可以修改 \<timeseries path> 为时间序列的完整路径。比如:
```

File: docs/Community-Project Committers.md [hunk 3]
```diff
@@ -42,3 +42,7 @@
 |  Tianan li     |  Tsinghua University |
 |  Yue Su        |	Tsinghua University |
 |  Hui Da        |	Lenovo              |
+|  Yuan Tian     |  Tsinghua University |
+|  Zesong Sun    |  Tsinghua University |
+|  Kaifeng Xue   |  Tsinghua University |
+|  Tianci Zhu    |                      |
```
Comment (Genius-pig) [inline docs/Community-Project Committers.md]: Do you forget @jack870131?

Comment (liutaohua) [reply docs/Community-Project Committers.md]: sorry，please give me more information about jack

Comment (qiaojialin) [reply docs/Community-Project Committers.md]: His name is Tsung-Han Tsai

File: docs/Documentation/UserGuide/3-Server/5-Docker Image.md [hunk 4]
```diff
@@ -22,7 +22,7 @@
 # Chapter 3: Server
 
 ## Docker Image
-Now a Dockerfile has been written at ROOT/docker/Dockerfile on the branch enable_docker_image.
+Now a Dockerfile has been written at ROOT/docker/Dockerfile under docker project.
```
Comment (qiaojialin) [inline docs/Documentation/UserGuide/3-Server/5-Docker Image.md]:
```suggestion
Now a Dockerfile has been written at docker/src/main/Dockerfile.
```

**Pull request merged**